public/sass/elements/_reset.scss

```diff
@@ -117,3 +117,7 @@ td {
   line-height: inherit;
   font-weight: normal;
 }
+
+abbr[title], acronym[title] {
```

Comment:
This won't prevent anyone from styling it by adding a class, this will just override the browser's default behaviour - which is to add a dotted underline or border.

eg `<abbr title="Web Content Accessibility Guidelines">WCAG</abbr>`

The title attribute is optional. 
I'm not sure we'd want abbreviated words without a title to be underlined either.

Normalize does this:
https://github.com/necolas/normalize.css/blob/master/normalize.css#L115

```
/**
 * 1. Remove the bottom border in Firefox 39-.
 * 2. Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari.
 */

abbr[title] {
  border-bottom: none; /* 1 */
  text-decoration: underline; /* 2 */
  text-decoration: underline dotted; /* 2 */
}

```

.

@robinwhittleton, should we also remove the border from abbr without an title and remove the bottom border as Normalize also does for older versions of Firefox?